plan/plans/show_test.go

```diff
@@ -11,4 +11,103 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package plans
+package plans_test
+
+import (
+	"fmt"
+
+	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb"
+	"github.com/pingcap/tidb/expression/expressions"
+	"github.com/pingcap/tidb/kv"
+	"github.com/pingcap/tidb/plan/plans"
+	"github.com/pingcap/tidb/sessionctx/variable"
+	"github.com/pingcap/tidb/stmt"
+)
+
+type testShowSuit struct {
+	txn  kv.Transaction
+	vars map[string]interface{}
+}
+
+// implement Context interface
+func (p *testShowSuit) GetTxn(forceNew bool) (kv.Transaction, error) { return p.txn, nil }
```

Comment:
For show statement, we need to access storage and will encounter error if use the mock context.